.github/workflows/ci-synthetic-test-runner.yml

```
@@ -0,0 +1,150 @@
+              repo: context.repo.repo,
+              sha: process.env.PR_SHA,
+              state: 'error',
+              context: '${{ steps.metadata.outputs.RUN_CONTEXT }}',
+              description: 'Synthetic test was cancelled',
+              target_url: `https://github.com/${context.repo.owner}/${context.repo.repo}/actions/runs/${process.env.GITHUB_RUN_ID}`
+            });
+
+      - name: Retrigger synthetic test 
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
```

actionlint (1.7.7)

property "check" is not defined in object type {metadata: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)